app/src/main/res/values/strings.xml

```diff
@@ -192,7 +192,7 @@
     </string>
     <string name="notifications_event_alarm_channel_name">Alarms for events</string>
     <string name="notifications_event_alarm_content_text">Alarm for event</string>
-    <string name="notifications_event_alarm_content_title">@string/app_name</string>
+    <string name="notifications_event_alarm_content_title" translatable="false">@string/app_name</string>
```

Comment:
The problem with having these translatable is that translators will translate the resource identifiers as well to something non-existent. It actually happened for me.

Reply:
Thank you for the translations. What do you suggest with regards to the resource identifiers?

Reply:
My simple fix was to mark these references as non-translatable like done here in this MR. You could also just copy/duplicate the strings and allow them to be translated differently. Either way, a resource reference shouldn't be translatable.

Reply:
Thank you. I applied your suggestion in 3d5bcd82e5b13e87ef60b94b2b4514050b4c588e.